databricks/koalas/frame.py

```diff
@@ -1467,6 +1467,101 @@ def shape(self):
         """
         return len(self), len(self.columns)
 
+    def merge(self, right: 'DataFrame', how: str = 'inner', on: str = None,
+              suffixes: Tuple[str, str] = ('_x', '_y')) -> 'DataFrame':
+        """
+        Perform a database (SQL) merge operation between two DataFrame objects.
```

Comment:
Can we inline the doc to Pandas'?

Reply:
Does it work for you now after 5472af1 and dde5b9c?